frontend/src/components/input-output/configure-ds/ConfigureDs.jsx

```diff
@@ -82,6 +82,7 @@ function ConfigureDs({
   };
 
   const handleTestConnection = (updatedFormData) => {
+    console.log("HERE!!!");
```

Comment:
Remove this log.

Reply:
@Deepak-Kesavan Thanks for the catch. Done.